components/src/Bloomreach/bloomreach.headersearch.main.less

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+ @import './../../../app/src/theme/common.less';
+ 
+  .main-search-container-br {
+    width: 170px;
+    text-align: right;
+  
+    .search-form-br {
+      position: relative;
+      display: inline-block;
+      
+      &:focus-within {
+        .input-search-br {
+          width: 270px;
```

Comment:
The width of this search box is fairly large. I'd suggest using 170px as the max width for the box and dropdown.

Reply:
I increased the width here as some of the suggestions that are coming back from bloomreach are 3 words long and dont fit within the 170px width.  Take a look at this screen shot here:  
![Screen Shot 2019-09-16 at 4 11 24 PM](https://user-images.githubusercontent.com/25829859/64999473-cac64500-d89c-11e9-9ba5-79889026d12a.png)

the 170px width would either cut off part of the suggestion or have it wrap around on the bottom which didn't look great.  Would be open to other suggestions with handling large recommendation strings in the suggestion box

Reply:
@aChanEP They should be wrapping in the container. 

I'd prefer either of the two options:

1) Increase the width from 270px to 275px or something higher so there's no buttons peeking out from under the search bar like below:
![image](https://user-images.githubusercontent.com/39800563/65042451-dc542f00-d926-11e9-858a-6f2b3933e6a8.png)

2) Decrease the width to 170px and force the suggestions to text-wrap in the container as below:

![image](https://user-images.githubusercontent.com/39800563/65042508-f8f06700-d926-11e9-9646-fc07b6877f04.png)

Reply:
Ok I think i will go with option 1 here.  Good catch with the icons not being fully covered.  Weirdly, seems to only be happening for me on the firefox.  No issues with styling on chrome.

Reply:
This issue should be fixed for firefox now